addons/udes_stock/data/picking_print.xml

```diff
@@ -11,6 +11,15 @@
             <field name="strategy_id" ref="model_udes_stock_stock_picking_print_strategy"/>
         </record>
 
+        <!-- Print on Picking Update event -->
```

Comment:
Bare with me, as I'm not very familiar with this...
When a picking is finished, the last thing the frontend invokes is `pick::update` that ends up calling `StockPicking::update_picking` where the print operation is implemented.
Does it means that we would have two printings due to the final `picking_update` and the above `picking_done`?

Reply:
This is the desired behaviour, eg. on the update_picking call we are printing package labels and the picking_done call we could (not for HDS) be printing a summary of that entire picking.